File: MPI/mpicheck/tests/functions/2.0/MPI_File_write_at_all_begin.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER(KIND=MPI_OFFSET_KIND) var_1
       TYPE(INTEGER) var_2(10)
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
        call mpi_file_write_at_all_begin(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_file_write_at_all_begin(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_File_write_at_all_begin.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_File var_0;
    MPI_Offset var_1;
    const void *var_2;
    int var_3;
    MPI_Datatype var_4;
    int ret;
    /* calls */
    ret = MPI_File_write_at_all_begin(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_File_write_at_all_begin(var_0, var_1, var_2, var_3, var_4);
    return 0;
}
